health-services/project/src/main/java/org/egov/project/repository/rowmapper/ProjectStaffRowMapper.java

```diff
@@ -0,0 +1,41 @@
+package org.egov.project.repository.rowmapper;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import digit.models.coremodels.AuditDetails;
+import org.egov.project.web.models.AdditionalFields;
+import org.egov.project.web.models.ProjectStaff;
+import org.springframework.jdbc.core.RowMapper;
+
+import java.sql.ResultSet;
+import java.sql.SQLException;
+
+public class ProjectStaffRowMapper implements RowMapper<ProjectStaff> {
+
+    private final ObjectMapper objectMapper = new ObjectMapper();
+
+    @Override
+    public ProjectStaff mapRow(ResultSet resultSet, int i) throws SQLException {
+        try {
+            return ProjectStaff.builder()
+                    .id(resultSet.getString("id"))
+                    .tenantId(resultSet.getString("tenantid"))
+                    .projectId(resultSet.getString("projectId"))
+                    .userId(resultSet.getString("staffId"))
+                    .startDate(resultSet.getLong("startDate"))
+                    .endDate(resultSet.getLong("endDate"))
+                    .additionalFields(objectMapper.readValue(resultSet.getString("additionalDetails"), AdditionalFields.class))
```

Comment:
You will have to write some conversion logic(null check) for additionalDetails. Refer to dev branch.